compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/CallableReferenceLowering.kt

```diff
@@ -282,16 +309,15 @@ internal class CallableReferenceLowering(private val context: JvmBackendContext)
                             //     f(::h) == g(::h)
                             //
                             parameter.isVararg && unboundIndex < argumentTypes.size && parameter.type == valueParameters[unboundIndex].type ->
-                                irArray(parameter.type) { addSpread(irGet(valueParameters[unboundIndex++])) }
+                                irGet(valueParameters[unboundIndex++])
```

Comment:
Why spread is removed?

Reply:
Because the current backend fails to copy the array here. :(

I also think that this is not the behavior we want, but technically it would be a breaking change and needs to go through the language committee. I'll file an issue about it later.

Reply:
Than 'In all other cases, excess arguments are packed into a new array' comment is obsolete

Reply:
This part of the comment refers to the next case. I've changed it to be less confusing.